code/game/machinery/vending.dm

```diff
@@ -2029,12 +2029,14 @@
 		/obj/item/weapon/reagent_containers/food/drinks/filk = 10,
 		/obj/item/weapon/reagent_containers/food/drinks/soda_cans/grifeo = 10,
 		/obj/item/weapon/reagent_containers/food/drinks/mannsdrink = 10,
+		/obj/item/weapon/reagent_containers/food/drinks/soda_cans/sportdrink = 10,
 		)
 	prices = list(
 		/obj/item/weapon/reagent_containers/food/drinks/groans = 20,
 		/obj/item/weapon/reagent_containers/food/drinks/filk = 20,
 		/obj/item/weapon/reagent_containers/food/drinks/soda_cans/grifeo = 30,
 		/obj/item/weapon/reagent_containers/food/drinks/mannsdrink = 10,
+		/obj/item/weapon/reagent_containers/food/drinks/soda_cans/sportdrink = 50,

```

Comment:
Would be easier (and make more sense?) to add a Brawndo Recharge Pack a-la the Boda vendor that has nothing but Brawndo. Not really within scope of the PR though.